upt/atcoder/__init__.py

```diff
@@ -1,96 +1,49 @@
-import argparse
-import logging
 import re
-import sys
 
-from ..util import Util, Driver, By, Keys
-from ..util.loginmanager import LoginManager
-from ..util.pathparser import PathParser
+from ..util import parser_common
+from ..util.parser_common import By
 
-logger = logging.getLogger("atcoder")
 
+def __login_checker(driver):
+    parser_common.wait_until(driver, By.CSS_SELECTOR, ".alert")
+    alert = driver.find_element_by_css_selector(".alert")
+    assert "Welcome" in alert.text, "Login failed"
```

Comment:
It is better to return a boolean here. Then raise a error if failed in `parser_common`.